platforms/android/build_sdk.py

```diff
@@ -487,7 +505,9 @@ def get_ndk_dir():
         builder.build_library(abi, do_install, args.no_media_ndk)
 
     builder.gather_results()
-
+   
+    check_have_ipp_flag(os.path.join(builder.libdest, "CMakeVars.txt"))
```

Comment:
I propose to add command line option, e.g. "--strict-dependencies" to make the check optional.